src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/helper/builders/Person.kt

```diff
@@ -164,6 +171,11 @@ class PersonBuilder(
     keepBiometrics = keepBiometrics,
   )
     .let { repository.save(it) }
+    .also {
+      if (whenCreated != null) {
```

Comment:
we have to updated this manually using SQL since in JPA it is read-only generated field